### PR TITLE
chore: Add Fried Frank message (ACC-300)

### PR DESCRIPTION
**Pull request by tolujimoh**
### Description
Onsite Cancellation Message for Fried Frank Company

### Screenshot

Screenshot 2020-04-28 at 17 31 55

### Related PR
[`next-ammit-api` - PR](https://github.com/Financial-Times/next-ammit-api/pull/663)

### CODE REVIEW

File: server/templates/partials/bottom/fried-frank-licence.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<div class="n-messaging-client-messaging-banner">
+	{{#> n-messaging-client/server/templates/components/o-banner
```
Comment (AniaMakes) [inline server/templates/partials/bottom/fried-frank-licence.html:2]: Is this a stray comment or is it actually doing anything?

File: server/templates/partials/bottom/fried-frank-licence.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+				<h1>Fried, Frank, Harris, Shriver & Jacobson LLP Cancellation</h1>
+			</header>
+			<p>
+				Fried, Frank, Harris, Shriver & Jacobson LLP has decided not to renew their corporate subscription to FT.com. If you have any questions about this please email
+				<a href="mailto:FFHSJResearchServices@friedfrank.com">FFHSJResearchServices@friedfrank.com</a>.
+			</p>
+		</div>
+		<div class="o-banner__actions">
+			<div class="o-banner__action">
+				<a href="mailto:FFHSJResearchServices@friedfrank.com" class="o-banner__button">
```
Comment (AniaMakes) [inline server/templates/partials/bottom/fried-frank-licence.html:19]: I'm guessing this is how it always worked, but if I clicked on the Keep My Access button I'd be quite surprised if I was taken to an email client.

File: server/templates/partials/bottom/fried-frank-licence.html (hunk 1 above)
Comment (tolujimoh) [reply server/templates/partials/bottom/fried-frank-licence.html:2]: That is how to render a partial block with Handlebars. The line renders `n-messaging-client/server/templates/components/o-banner` template.

Have a look at the [Handlebars Documentation](https://handlebarsjs.com/guide/partials.html#partial-blocks)

File: server/templates/partials/bottom/fried-frank-licence.html (hunk 2 above)
Comment (tolujimoh) [reply server/templates/partials/bottom/fried-frank-licence.html:19]: I believe it is usually a CTA link, however, the Client is sending their own internal communication about the cancellation.

Comment (umbobabo) [inline server/templates/partials/bottom/fried-frank-licence.html:19]: Why not automatically add a subject?
```suggestion
				<a href="mailto:FFHSJResearchServices@friedfrank.com?subject=Keep my access to FT.com" class="o-banner__button">
```
Not our call but maybe you can suggest it to UXD.

**Pull request merged**